canonical-websites/build.snapcraft.io
# Vanilla list component

## Done

Added new List component with Vanilla based styles and custom green tick style.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Landing page should have a list with vanilla styles and custom green ticks.


## Issue / Card

Fixes #891 

## Screenshots

<img width="1046" alt="screen shot 2017-07-24 at 10 00 51" src="https://user-images.githubusercontent.com/83575/28513589-03d0a33c-7057-11e7-8afd-6d2146409428.png">
